src/shared/nt-features/CreateFeatureForm.react.js

```diff
@@ -65,19 +69,26 @@ class CreateFeatureForm extends React.Component {
   }
 
   handleSubmit = e => {
+    e.preventDefault()
```

Comment:
Why `e.preventDefault()` before mutate`? (Really don't know if the order matter)